src/main/java/net/fabricmc/stitch/util/EnvironmentAnnotations.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.stitch.util;
+
+/**
+ * Annotations used to declare a sided environment.
+ */
+public class EnvironmentAnnotations {
```

Comment:
You know what Zidane would say here...

~~final and private ctor~~

Reply:
i mean he'd probably say "tsk tsk" :p (but note taken)